x-pack/plugins/apm/server/routes/metrics/by_agent/serverless/cold_start_duration.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { FAAS_COLDSTART_DURATION } from '../../../../../common/elasticsearch_fieldnames';
+import { Setup } from '../../../../lib/helpers/setup_request';
+import { fetchAndTransformMetrics } from '../../fetch_and_transform_metrics';
+import { ChartBase } from '../../types';
+
+const chartBase: ChartBase = {
+  title: i18n.translate('xpack.apm.agentMetrics.serverless.coldStart', {
+    defaultMessage: 'Cold start',
+  }),
+  key: 'cold_start',
+  type: 'linemark',
+  yUnit: 'number',
```

Comment:
If we convert the values to microsecond (currently it's milliseconds), we can use the `time` unit here, right?

Reply:
we can use `time` here even with the value in mmilliseconds